content/javascript/concepts/dates/terms/toLocaleString/toLocaleString.md

```diff
@@ -0,0 +1,89 @@
+---
+Title: 'toLocaleString Method'
+Description: 'A method to format a Date object as a string, based on the locale.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+  
+Tags:
+  - 'JavaScript'
+  - 'Methods'
+  - 'Date'
+  
+CatalogContent:
+  - 'learn-javascript'
+  - 'paths/web-development'
+---
+
+# toLocaleString() Method :
+
+The `.toLocaleString()` method in JavaScript is used to format a [`Date` object](https://www.codecademy.com/resources/docs/javascript/dates), as a string based on the specified locale. This means it takes into account cultural settings such as `language` and `date/time` formatting preferences specific to the selected region or country.
+
+## Syntax : 
+
+```js {copy}
+toLocaleString(locales [, options])
+```
+- `locales` : Represents the `locale or locales` to use when formatting the `date`.
+- `options` : Contains additional formatting options for the `date`, such as the format of the `weekday`, `month`, `year`, etc.
+
+
+## Example : 
+
+```js {copy}
+const date = new Date(); // Current date and time
+const options = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' };
+console.log(date.toLocaleString('en-US', options));
+```
```

Comment:
Please modify this example, the code has been plagiarized.
Plagiarised from: `https://debbie.codes/blog/date-formatting/`